imports/plugins/core/router/lib/router.js

```diff
@@ -35,12 +34,41 @@ if (Meteor.isClient) {
 
 // Base router class (static)
```

Comment:
If you could change the syntax here to jsdoc style: `/** Class representing a static base router */`, jsdoc will render the rest of the class.